snapshot/lib/snapshot/runner.rb

```diff
@@ -108,7 +108,7 @@ def copy_simulator_logs(device_name, language, locale, launch_arguments)
 
       UI.header("Collecting system logs #{device_name} - #{language}")
       logarchive_dest = File.join(language_folder, "system_logs-" + Digest::MD5.hexdigest(components.join("-")) + ".logarchive")
```

Comment:
Well, my plan is to make a _real_ logarchive that is just the name as you have here. Not make an additional folder to hold just `system_logs.logarchive`